src/test/resources/config/application.yml

```diff
@@ -52,6 +52,9 @@ spring:
             hibernate.cache.use_query_cache: false
             hibernate.generate_statistics: true
             hibernate.hbm2ddl.auto: validate
+            hibernate.model.generator_name_as_sequence_name: false # see org.hibernate.cfg.AvailableSettings.PREFER_GENERATOR_NAME_AS_DEFAULT_SEQUENCE_NAME
```

Comment:
I think this properties also must be added to 
src/resources/config/application.yml

need to check issue with sequence in dev and prod profile

Reply:
already added to dev, prod